lib/Runtime/PlatformAgnostic/Platform/Linux/DateTime.cpp

```diff
@@ -13,155 +13,219 @@ namespace PlatformAgnostic
 namespace DateTime
 {
 
-    #define updatePeriod 1000
-
-    const WCHAR *Utility::GetStandardName(size_t *nameLength) {
-        data.UpdateTimeZoneInfo();
-        *nameLength = data.standardZoneNameLength;
-        return data.standardZoneName;
+    static inline bool IsLeap(const int year)
+    {
+        return (0 == (year & 3)) && (0 != (year % 100) || 0 == (year % 400));
```

Comment:
Instead of year & 3, can you do year % 4? It makes it more readable and the compiler should take care of optimizing it to & 3